### DISCUSSION TITLE
Can't add extra components to existing install

### DISCUSSION THREAD

**Discussion by rgstephens** [fluxcd/flux2#1203] [Q&A]
I'm running `0.10.1` and want to try the new [image automation](https://particule.io/en/blog/flux-auto-image-update/) feature.

When I re-run the bootstrap command, I get the error `cluster already bootstrapped to ...` and nothing is added to my repo.

I'm re-running the same command I used for the initial setup and adding the `--components-extra` option.

```sh
flux bootstrap github \
  --owner=$GITHUB_USER \
  --repository=flux-test \
  --branch=main \
  --path=./mycluster \
  --components-extra=image-reflector-controller,image-automation-controller \
  --personal
```

**Comment by stefanprodan** [comment]
As the error message says you need to use the same path as your original bootstrap. Changing the path on upgrades is not supported as it will wipe clean you cluster.

**Reply by rgstephens** [reply]
I'm using the same path as the original bootstrap.

**Comment by oliver-kriska** [comment]
I think you should use install command not bootstrap, so for example:
```bash 
flux install \
          --components-extra=image-reflector-controller,image-automation-controller \
          --export > ./mycluster
```

**Reply by thakurnishu** [reply]
Guys, after getting this file, paste its contents into `gotk-components.yaml`

**Reply by Notliam99** [reply]
Note Works also updated FluxCD 👍🏿
